### PR TITLE
Feature: ClamScan container config and scan file.

### PR DESCRIPTION
**Pull request by sfaulkner89**

### CODE REVIEW

File: packages/web-service/src/services/virus-scan.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    active: true
+  },
+  preference: 'clamdscan'
+}
+const ClamScan = new NodeClam().init(options)
+
+export async function scanFile (filename) {
+  return ClamScan.then(async (clamscan) => {
+    try {
+      const dir = `../..${process.env.SCANDIR}/${filename}`
```
Comment (GrahamMWillis) [inline packages/web-service/src/services/virus-scan.js:21]: This needs to be an absolute path I think

File: docker/cloud-services.yml [hunk 2]
```diff
@@ -68,3 +68,25 @@ services:
...
+    deploy:
+      restart_policy:
+        condition: on-failure
+    volumes:
+      - virus-db:/var/lib/clamav
+      - scandir:/scandir
+
+volumes:
+  virus-db:
+  scandir:
```
Comment (GrahamMWillis) [inline docker/cloud-services.yml:92]: Not a big deal but the convention is a CR at the EOF

**Review by GrahamMWillis — CHANGES_REQUESTED** [review]

**Comment by sfaulkner89** [comment]
After talking to Kealan we decided that if the scan fails, the file should be deleted on my end for consistency's sake. I've added that in, fixed your comments and the code smells.

File: packages/web-service/src/services/virus-scan.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    timeout: parseInt(process.env.CS_TIMEOUT),
+    multiscan: false,
+    active: true
+  },
+  preference: 'clamdscan'
+}
+const ClamScan = new NodeClam().init(options)
+
+export async function scanFile (filename) {
+  const dir = `.${process.env.SCANDIR}/${filename}`
```
Comment (GrahamMWillis) [inline packages/web-service/src/services/virus-scan.js:19]: I still think this should be an absolute path - do you want to have a chat about it?

**Review by GrahamMWillis — CHANGES_REQUESTED** [review]

**Pull request merged**